elements/pfe-sass/extends/_typography.scss

```diff
@@ -0,0 +1,83 @@
+%pfe-headline-xxl {
+    font-family: var(--#{$repo}-theme--font-family--heading, #{pfe-var(font-family--heading)} );
```

Comment:
This is duplicative as pfe-var outputs the entire var statement with the empty hook at the front: `font-family: pfe-var(font-family--heading);`

Reply:
right you are, thank you! I removed the duplicates from font-family and font-size. Does it make sense to leave the greek names along with the original PFE names in the stack for the weight and line height?
 
![image](https://user-images.githubusercontent.com/456863/96736198-5bb5fc80-138a-11eb-8e5a-43852adf9258.png)

Reply:
I think so - it kind of replicates our "empty local variables" stack inside components.